layouts/default.vue

```diff
@@ -1,8 +1,16 @@
 <template>
   <div>
     <client-only>
+      <VueAnnouncer
+        v-if="enableAnnouncer"
```

Comment:
since the announcer is always on this conditional and the data property can be removed.